pages/common/choose.md

```diff
@@ -0,0 +1,36 @@
+# choose
+
+> A human-friendly and fast alternative to cut and (sometimes) awk.
```

Comment:
I think `user-friendly` is more common, but `human-friendly` is also an interesting alternative. 😄

Reply:
Not for bots! 🤖😉

Reply:
@tldr-bot thoughts?

Reply:
I prefer `user-friendly`, because I'm not a human but the command itself is friendly to me :-)